### ISSUE TITLE
"Asks" feature

### ISSUE THREAD

**Issue by danimesq** [mkg20001/ZeroMe#57]
A user can publish a "ask" in another profile, then the user will answer.

**Comment by mkg20001** [comment]
This is q&a, not social media
Instead there should be a ZeroNet version of Quora or StackOverflow

**Comment by danimesq** [comment]
This idea is inspirated by Ask.fm social media. A lot of casual users will like more ZeroMe if this function is active.

**Comment by mkg20001** [comment]
Yes but we don't want to turn ZeroMe into something that has so many features that nobody wants to use it anymore.
See what happened after WhatsApp added stories?

**Comment by danimesq** [comment]
Its hard to think people will stop making asks. Even in ZeroMe.

**Comment by mkg20001** [comment]
Ok, let's see. We can add an option to disable it.

**Comment by danimesq** [comment]
Moved to: https://github.com/World-wd/Peeper/issues/53.
